Library/Homebrew/config.rb

```diff
@@ -27,7 +27,11 @@ def mkpath
 HOMEBREW_CACHE_FORMULA = HOMEBREW_CACHE+"Formula"
 
 unless defined? HOMEBREW_BREW_FILE
```

Comment:
Although that makes me wonder if it's better to just set and/or check it in a single place?